etc/links.profile

```diff
@@ -0,0 +1,51 @@
+# Firejail profile for elinks
+# Description: Text WWW browser
+# This file is overwritten after every install/update
+#
+# Persistent local customizations
+include links.local
+# Persistent global definitions
+# include globals.local
+
+blacklist /tmp/.X11-unix
+
+noblacklist ${HOME}/.links
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.links
+whitelist ${HOME}/.links
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
```

Comment:
Does it work without including whitelist-common? That's mostly for things like icons and gnome/KDE-specific directories -- does links need to access these?

Reply:
Thanks, I will drop both `-common` and `-var-common` includes as apparently they're not needed

Reply:
with `include whitelist-var-common.inc` is more restrictive than without.

Reply:
@jose1711 Do keep `include whitelist-var-common.inc` :wink: